### DISCUSSION TITLE
DeepMind又贏了！AI天氣象預報準確度遠超氣象台

### DISCUSSION THREAD

**Discussion by ccathyy** [1101-GS3073-C/Python-Community-GS3073-C#102] [General]
因為選了這門通識課而讓我會特別留意”AI”這個詞。
近期看了許多報導，發現在我們日常生活中，AI無所不在!
讚嘆科技進步的同時，我在思索怎麼做才能讓自己更進步，才能從眾人裡脫穎而出。
未來想進科技公司工作，想當工程師的我，必定要認真習得一些基本的程式語言，還要保有創新思維、軟實力，以及獨立思考的能力。
好多能力需要好好培養起，我要持續加油，持續進步!

今日閱讀的新聞的出處: https://www.techbang.com/posts/90385-deepmind-wins-again-ai-weather-forecast-hangs-weather-station

**Comment by Timmy110403540** [comment]
準確的預報天氣，得以更精準地知悉當天的狀況!

**Comment by chia-kai-chang** [comment]
我認為這是AI的智慧價值。

**Comment by ghost** [comment]
以前小時候總有人說大自然深不可測，現在看來已經可以被慢慢摸透了，覺得未來一定會更有幫助

**Comment by Shiba010** [comment]
預測這個領域本就是ai擅長的範圍，看起來以後氣象台也要應用ai來預測天氣了。

**Comment by Lynn12378** [comment]
AI真的帶來很多方便，但也讓越來越多的職業被取代
